client/tx/factory.go

```diff
@@ -254,12 +276,27 @@ func (f Factory) PrintUnsignedTx(clientCtx client.Context, msgs ...sdk.Msg) erro
 		_, _ = fmt.Fprintf(os.Stderr, "%s\n", GasEstimateResponse{GasEstimate: f.Gas()})
 	}
 
-	tx, err := f.BuildUnsignedTx(msgs...)
+	unsignedTx, err := f.BuildUnsignedTx(msgs...)
 	if err != nil {
 		return err
 	}
 
-	json, err := clientCtx.TxConfig.TxJSONEncoder()(tx.GetTx())
+	tipAmt := f.fees
+	if f.tip != nil {
+		tipAmt = f.tip.Amount
+	}
+	if f.SignMode() == signing.SignMode_SIGN_MODE_AMINO_AUX {
+		unsignedTx.SetTip(&tx.Tip{
+			Amount: tipAmt,
```

Comment:
we should always set `Tipper: clientCtx.FromAddress.String()` for simplicity.

Yeah AMINO_AUX's sign doc doesn't need Tipper field, but it's still useful in the code to put it everywhere.